### ISSUE TITLE
License of exercism icon / using it when linking to exercism

### ISSUE THREAD

**Issue by SaschaMann** [exercism/exercism#4673]
Hello,

I link to exercism on my website and would like to use the new icon for that, but I'm unsure about its license and under what conditions I may use it.

The logo itself is in [exercism/website](https://github.com/exercism/website/blob/master/app/assets/images/icon.png), which is published under AGPL-3.0. This is primarily a license for software/source code, and its conditions seem unusual for an icon, hence I'm asking here.

Is it allowed to use the icon on one's website when linking to exercism, what conditions need to be met and how to properly give credit?

**Comment by iHiD** [comment]
Hey! We need to put up a page that has the logo in with some guidance for this, and firm up our actual guidance in the process, including the licence. Thank you for the prompt.

> Is it allowed to use the icon on one's website when linking to exercism, what conditions need to be met and how to properly give credit?

Exercism grants you personally the (revokable) right to use the Exercism logo on your website as long as you link back to https://exercism.io from it. I recommend using the icons from [here](https://github.com/exercism/website/tree/master/app/assets/images/favicon) until we publish some more high-res ones.

We'll firm this up into a licence that other people can use too in the coming weeks :)

**Comment by SaschaMann** [comment]
Sounds good, thanks!

**Comment by ErikSchierboom** [comment]
Hello 👋

With the launch of Exercism v3, we are **closing all issues in this repository** to help give us a clean slate to detect new problems. If this issue is still relevant to Exercism v3 (e.g. it's a feature that we haven't implemented in v3, or a bug that still exists), please reopen it and we will review it and post an update on it as soon as we get chance.

Thanks for helping make Exercism better, and we hope you enjoy v3 🙂

**Comment by SaschaMann** [comment]
AFAICT this is still relevant and should be reopened. But I'm not entirely sure. @iHiD has this been clarified in docs somewhere?

**Comment by iHiD** [comment]
No. Probably not. We should probably add a file to https://exercism.org/docs/using/legal that is about Exercism's trademarks etc.
